tasks/config.yml

```diff
@@ -293,3 +293,27 @@
     dest: "{{ bbb_nginx_root | regex_replace('\\/$', '') }}/{{ bbb_custom_presentation_name | default(bbb_custom_presentation) }}"
     mode: '0644'
   when: bbb_custom_presentation is defined
+
+  - name: Set the retention period for presentations, red5, kurento, and freeswitch caches
+  lineinfile:
```

Comment:
Instead of working with so many `lineinfile` replacement and RegEx I would prefer starting to template this file.

Reply:
I don't really know what the best practice for templating this file would be. 
Is it just to take [the file from bbb](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-config/cron.daily/bigbluebutton) and template the vars we want to set dynamically? If so, I would also create a new directory `bbb-config/cron.daily`.

Reply:
`templates/bbb-config/cron.daily` would be OK for me.
Please add the URL to the file into the template as a comment.
```
# https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bbb/2.3/tasks/kurento.yml
```
```
history=5
unrecorded_days=14
published_days=14
log_history=28
```
For now I think only this part needs to be templated.
Something like this - open for others vars
```
history={{ bbb_cron_history }}
unrecorded_days={{ bbb_cron_unrecorded_days }}
published_days={{ bbb_cron_published_days }}
log_history={{ bbb_cron_log_history }}
```
And using the defaults from the file as default in the vars.